packages/backend-core/src/queue/inMemoryQueue.ts

```diff
@@ -176,7 +186,7 @@ class InMemoryQueue implements Partial<Queue> {
 
   async removeRepeatableByKey(id: string) {
     for (const [idx, message] of this._messages.entries()) {
-      if (message.opts?.jobId?.toString() === id) {
+      if (message.id === id) {
```

Comment:
In `getRepeatableJobs` we transform each message to a `JobInformation` object. This sets `.key` to be the same as `.id`. We're relying on that fact here, as messages are not stored with a `.key` property.